main.go

```diff
@@ -87,11 +87,13 @@ func main() {
 func parseConfig() (spirev1alpha1.ControllerManagerConfig, ctrl.Options, []*regexp.Regexp, error) {
 	var configFileFlag string
 	var spireAPISocketFlag string
+	var expandEnvFlag bool
 	flag.StringVar(&configFileFlag, "config", "",
 		"The controller will load its initial configuration from this file. "+
 			"Omit this flag to use the default configuration values. "+
 			"Command-line flags override configuration from this file.")
 	flag.StringVar(&spireAPISocketFlag, "spire-api-socket", "", "The path to the SPIRE API socket (deprecated; use the config file)")
+	flag.BoolVar(&expandEnvFlag, "expandEnv", false, "Expand environment variables in SPIRE Controller Manager config file")
```

Comment:
This component uses kebab-case for flags, i.e. `expand-env`

Reply:
Changed